test_bot/test_lichess.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for the lichess communication."""
+
+from lib import lichess
+import logging
+import os
+import pytest
+
+
+def test_lichess() -> None:
+    """Test the lichess communication."""
+    token = os.getenv("LICHESS_BOT_TEST_TOKEN")
+    if token is None:
+        pytest.skip("Lichess-bot test token must be set.")
+    li = lichess.Lichess(token, "https://lichess.org/", "0.0.0", logging.DEBUG, 3)
+    assert len(li.get_online_bots()) > 20

```

Comment:
Why 20? While there are usually a large number of bots online at any given time, I don't think we should be testing something that's out of our control. Maybe a test like this would be more reliable:
```python
online_bots = li.get_online_bots()
assert isinstance(list, online_bots)
assert any(filter(lambda bot: bot['username'] == "BadSunfish", online_bots))
```

Reply:
A problem is that it takes some time until bots show up in the `online_bots` list, so `assert any(filter(lambda bot: bot['username'] == "BadSunfish", online_bots))` will fail most of the times. It is quite hard to create a meaningful test that is completely under our control.